helsingborg-stad/helsingborg-io-sls-api
# Fix/viva case eventflow

## Explain the changes you’ve made
Added new state: VIVA_APPLICATION_LOCKED
Refactored tests

# How to test
1. Checkout branch
2. Deploy viva/microservice to your sandbox
3. Have no cases in your DynamoDB cases table for user 198602132394 (Bror Christiansson)
4. Verify that user 198602132394 can apply for recurring case in Viva
5. Run aws-vault to connect to your aws sandbox environment
6. Login with 198602132394 on the app and fulfill the application and submit
7. Complete the handling process and make the application status closed:???
